src/webviews/apps/settings/partials/commit-graph.html

```diff
@@ -64,6 +64,21 @@ <h2>
 						</div>
 					</div>
 
+					<div class="setting">
+						<div class="setting__input">
+							<label for="graph.scrollRowPadding">Start scrolling at </label>
+							<input
+								id="graph.scrollRowPadding"
+								name="graph.scrollRowPadding"
+								type="number"
+								placeholder="0"
+								data-setting
+								data-default-value="0"
+							/>
+							<label for="graph.searchItemLimit"> rows from the edge</label>
```

Comment:
Is this label meant to point to `graph.scrollRowPadding`?

Reply:
Oops! Yep, nice catch. Will update it. Thanks!